lib/propcheck/utils.ex

```diff
@@ -142,4 +142,60 @@ defmodule PropCheck.Utils do
   end
   defp parse_io_spec_format(char_or_spec), do: char_or_spec
 
+  def toplevels(graph) do
+    with(
```

Comment:
Are the brackets after the `with` required? Looks strange, but may be correct.

Reply:
Yes, they required for such formatting.